packages/http-client-python/generator/pygen/codegen/serializers/__init__.py

```diff
@@ -154,11 +154,13 @@ def serialize(self) -> None:
                 # add clients folder if there are clients in this namespace
                 self._serialize_client_and_config_files(client_namespace, client_namespace_type.clients, env)
             else:
-                # add pkgutil init file if no clients in this namespace
-                self.write_file(
-                    exec_path / Path("__init__.py"),
-                    general_serializer.serialize_pkgutil_init_file(),
+                # add/keep pkgutil init file if no clients in this namespace
+                init_path = exec_path / Path("__init__.py")
+                original_init_content = self.read_file(init_path)
+                init_content = (
+                    original_init_content if original_init_content else general_serializer.serialize_pkgutil_init_file()
                 )
+                self.write_file(init_path, init_content)
```

Comment:
I thought about other solutions like add judge logic about whether namespace contains `_generated`. But we can't assume which kind of namespace users may set. Maybe they want namespace like `azure.test.generated` without `_` then the judge logic can't work again.

So the best solution is we don't overwrite the existing pkgutil init file: 
- If  `__init__` is generated by codegen, no need to overwrite it with same content
- If `__init__` is written by users, leave it unchanged.